tests/acceptance/new-test.js

```diff
@@ -540,6 +541,27 @@ describe('Acceptance: ember new', function () {
       checkFileWithEmberCLIVersionReplacement(fixturePath, 'package.json');
     });
 
+    if (isExperimentEnabled('PNPM')) {
+      it('app + pnpm + welcome', async function () {
+        await ember(['new', 'foo', '--skip-npm', '--skip-git', '--pnpm']);
+
+        let fixturePath = 'app/pnpm';
+
+        ['app/templates/application.hbs', '.github/workflows/ci.yml', 'README.md'].forEach((filePath) => {
+          checkFile(filePath, path.join(__dirname, '../fixtures', fixturePath, filePath));
+        });
+
+        expect(file('.travis.yml')).to.not.exist;
```

Comment:
I know this is copied from other tests, but this assertion seems weird considering we have removed travis as an option afaik

Reply:
Seems like it should be done in another PR, as we don't want to mix purposes of PRs.
If we're going to remove travis, it should be done full-assedly.

Since the file still exists in the blueprint, i think it makes sense to include the assertion that travis is not present by default in the output

Reply:
Ok if it still exists in the blueprint. I think I was misremembering Bert removing travis support completely.